schema.d.ts

```diff
@@ -250,17 +254,15 @@ export interface CustomElement extends ClassLike {
    */
   customElement: true;
 
-  members?: Array<CustomElementMember>;
-  
+  // members?: Array<CustomElementMember>;
+
   /**
    * Whether the custom element is deprecated.
    * If the value is a string, it's the reason for the deprecation.
    */
-  deprecated?: boolean|string;
+  deprecated?: boolean | string;
```

Comment:
`deprecated` is also declared in `ClassLike`.

Reply:
removed